styles/RadioButtons.module.css

```diff
@@ -14,7 +14,8 @@
 }
 
 .button {
-  display: none;
+  width: 0;
+  visibility: hidden;
```

Comment:
You still need to show the `<input>` elements otherwise they are not accessible to the keyboard or screen readers.